tools/webots.py

```diff
@@ -41,6 +42,10 @@ def register(command):
         help="Disables cleaning the build volume before compiling",
     )
 
+    build_subcommand.add_argument(
+        "-j", "--jobs", dest="jobs", action="store", help="Dictates the number of jobs to run in parallel"
```

Comment:
https://blog.kitware.com/cmake-building-with-all-your-cores/
```suggestion
        "-j", "--jobs", dest="jobs", action="store", help=f"Dictates the number of jobs to run in parallel ({multiprocessing.cpu_count() + 2} by default"
```

Reply:
@Y0sh1dk I like this suggestion. It does make me consider if we should go further though, and tell people how we get the number, so that people can be sure it's different on different machines and what it will be. Basically, mention that it's number of threads plus 2, so we don't have the ambiguity that `ninja --help` introduces.